Vagrantfile

```diff
@@ -7,13 +7,17 @@ Vagrant.configure(2) do |config|
   config.vm.box = "ubuntu/trusty64"
 
   config.vm.synced_folder "~/.aws", "/home/vagrant/.aws"
-  config.vm.synced_folder "~/.ivy2", "/home/vagrant/.ivy2"
-
+  config.vm.synced_folder "./", "/vagrant", type: "rsync",
+    rsync__exclude: ["app-backend/**/target/",
```

Comment:
I think that we need to make sure that `.git` is ignored as well.

Reply:
Ah, I left it in there so `scripts/cibuild` and `scripts/test` didn't break but I guess people can just tag containers with `latest`.